apps/android/app/src/main/java/chat/simplex/app/SimplexApp.kt

```diff
@@ -173,7 +172,18 @@ class SimplexApp: Application(), LifecycleEventObserver {
       val s = Semaphore(0)
       thread(name="stdout/stderr pipe") {
         Log.d(TAG, "starting server")
-        val server = LocalServerSocket(socketName)
+        var server: LocalServerSocket? = null
+        for (i in 0..100_000 step 1234) {
```

Comment:
```suggestion
        for (i in 0..1000) {
```

I don't think step 1234 makes it any more random tbh, as it still scans the same numbers, it just makes it more confusing...

Reply:
also trying ~1000 sockets may be slow, not sure, reduced to 100...